packages/ember-glimmer/tests/integration/helpers/log-test.js

```diff
@@ -0,0 +1,60 @@
+import { RenderingTest, moduleFor } from '../../utils/test-case';
+import { get } from 'ember-metal/property_get';
+import Logger from 'ember-metal/logger';
+
+var originalLog, logCalls;
+
+moduleFor('Helpers test: {{log}}', class extends RenderingTest {
+
+  constructor() {
+    super();
+
+    originalLog = Logger.log;
+    logCalls = [];
+    Logger.log = function(...args) {
+      logCalls.push(...args);
+    };
+  }
+
+  teardown() {
+    Logger.log = originalLog;
+  }
+
+  assertLog(assert, values) {
+    this.assertText('');
+    assert.strictEqual(logCalls.length, values.length);
+
+    for (let i = 0, len = values.length; i < len; i++) {
+      assert.strictEqual(logCalls[i], values[i]);
+    }
+  }
```

Comment:
@chancancode - I think we could pass `assert` in `abstract-test-case.js` in the `moduleFor` function, when we new up `TestClass`. I can go ahead and add that if you'd like.